modules/contact/src/constraints/MechanicalContactConstraint.C

```diff
@@ -1688,6 +1712,24 @@ MechanicalContactConstraint::computeQpOffDiagJacobian(Moose::ConstraintJacobianT
   return 0.0;
 }
 
+Real
```

Comment:
It looks like we're always using the summation of `slaveGapOffset()` and `mappedMasterGapOffset()`. Why not just have a single function `gapOffset()` that returns the sum of the two?  That would make the code that uses this a little cleaner.  We also need to get rid of all usage of 'master' and 'slave' in the code, so that change would accomplish both goals.